scripts/build-dtest-jars.sh

```diff
@@ -27,7 +27,8 @@ mkdir -p "${BUILD_DIR}"
 
 # host key verification
 mkdir -p ~/.ssh
-ssh-keyscan github.com >> ~/.ssh/known_hosts
+REPO_HOST=$("${SCRIPT_DIR}/get-hostname.sh" "${REPO}")
```

Comment:
nit: instead of eval the script, it could source the script if the functionality is defined as a function.